Source/common/CertificateHelpers.m

```diff
@@ -0,0 +1,42 @@
+/// Copyright 2023 Google LLC
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     https://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+
+#include "Source/common/CertificateHelpers.h"
+
+#include <Security/SecCertificate.h>
```

Comment:
```suggestion
#import "Source/common/CertificateHelpers.h"

#import <Security/SecCertificate.h>
```

Reply:
Updated `CertificateHelpers.h` to be imported, but left `SecCertificate.h` as it's a C/C++ header